01week/rockPaperScissors.js

```diff
@@ -25,7 +41,7 @@ const rockPaperScissors = (hand1, hand2) => {
 function getPrompt() {
   rl.question('hand1: ', (answer1) => {
     rl.question('hand2: ', (answer2) => {
-      console.log( rockPaperScissors(answer1, answer2) );
+      console.log(rockPaperScissors(answer1.toLowerCase(), answer2.toLowerCase()));
```

Comment:
I would have put this in the rockPaperScissors() function, instead of here.
Also need to trim to handle spaces.